python/src/SobolIndicesAlgorithmImplementation_doc.i.in

```diff
@@ -395,7 +395,7 @@ OT_SobolIndicesAlgorithm_draw_doc
 // ---------------------------------------------------------------------
 
 %define OT_SobolIndicesAlgorithm_DrawCorrelationCoefficients_doc
-"Draw the correlation coefficients. 
+"Draw the correlation coefficients.
```

Comment:
This is the doc before the commit:
![image](https://user-images.githubusercontent.com/31351465/156365313-8aed4ca4-bfef-4f12-954b-af50453ca8ef.png)
The text should not be in bold. 
This change is unrelated to the current development.